### PR TITLE
Fix Chinese line break problem

### PR DESCRIPTION
**Pull request by lxwinspur**
When some buttons are described in Chinese, the shape of the button will change due to word wrapping.

Signed-off-by: George Liu <liuxiwei@inspur.com>

### CODE REVIEW

File: src/views/ResourceManagement/CapacityOnDemand/CapacityOnDemandActivation.vue [hunk 1]
```diff
@@ -46,7 +46,7 @@
                   </b-form-invalid-feedback>
                 </b-input-group>
               </b-form-group>
-              <b-col align-self="baseline">
+              <b-col sm="2" align-self="baseline">
```
Comment (Nikhil-Ashoka) [inline src/views/ResourceManagement/CapacityOnDemand/CapacityOnDemandActivation.vue:49]: The issue still occurs with different screen sizes.
image

Comment (lxwinspur) [reply src/views/ResourceManagement/CapacityOnDemand/CapacityOnDemandActivation.vue:49]: Thanks @Nikhil-Ashoka 
So, could you close the status bar on the right and check again?

Comment (Nikhil-Ashoka) [reply src/views/ResourceManagement/CapacityOnDemand/CapacityOnDemandActivation.vue:49]: Yes, It works fine for the full screen in my system. However, If we are using a smaller screen, This issue is recurring.

Comment (lxwinspur) [reply src/views/ResourceManagement/CapacityOnDemand/CapacityOnDemandActivation.vue:49]: What size screen did you use to test? Let's see if this problem is expected.

Comment (Nikhil-Ashoka) [reply src/views/ResourceManagement/CapacityOnDemand/CapacityOnDemandActivation.vue:49]: Eg: Responsive: 770*1280 to 830*1280. This issue has occurred again.

**Review by Nikhil-Ashoka — APPROVED** [review]
Looks Good, Thanks!!!

**Pull request merged**